code/game/objects/items/reagent_containers/hypospray.dm

```diff
@@ -138,6 +133,17 @@
 			playsound(user.loc, 'sound/weapons/thudswoosh.ogg', 25, 1, 7)
 			return FALSE
 
+		if(!M.can_inject(user, TRUE, user.zone_selected, TRUE))
+			return FALSE
+
+	var/list/injected = list()
+	for(var/datum/reagent/R in reagents.reagent_list)
+		injected += R.name
+	var/contained = english_list(injected)
+	log_combat(user, A, "injected", src, "Reagents: [english_list(reagents.reagent_list)]")
+
+	if(ismob(A))
+		var/mob/living/M = A
```

Comment:
You're checking for ismob but typecasting a living.

Reply:
Whops, thanks for spotting.